contributing/getting-started-e2e.md

```diff
@@ -153,21 +153,18 @@ Now that you have all of the files copied locally you can actually build the cod
 
 amphtml uses Node.js, the Yarn package manager and the Gulp build system to build amphtml and start up a local server that lets you try out your changes.  Installing these and getting amphtml built is straightforward:
 
-* Install [NodeJS](https://nodejs.org/) version >= 6 (which includes npm)
+* Install the LTS version of [NodeJS](https://nodejs.org/)
```

Comment:
Nit: "active LTS" vs "current LTS" (https://github.com/nodejs/Release#release-schedule1) though we should probably just specify Node 6+. Node 8 will be "active LTS" next month.

Reply:
Fixed.